bin/phpbench.bat

```diff
@@ -0,0 +1,2 @@
+@echo off
+php.exe "%~dp0\phpbench.php" %*
```

Comment:
@WouterJ that only works if you use it as a library, composer doesn't create those for standalone projects